integration/singularity/reader.go

```diff
@@ -0,0 +1,76 @@
+package singularity
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+
+	singularityclient "github.com/data-preservation-programs/singularity/client/swagger/http"
+	"github.com/data-preservation-programs/singularity/client/swagger/http/file"
+	"github.com/gotidy/ptr"
+)
+
+// io.ReadSeekCloser implementation that reads from remote singularity
+type SingularityReader struct {
+	client *singularityclient.SingularityAPI
+	fileID uint64
+	offset int64
+	size   int64
+}
+
+func (r *SingularityReader) Read(p []byte) (int, error) {
+	logger.Infof("buffer size: %v", len(p))
```

Comment:
* Change to debug level as this can be noisy?
* Use `w` variation instead of format for nicer JSON log formatting.

Reply:
oops, this was supposed to be removed. i will remove it.